### ISSUE TITLE
[ADE 508 Audit]: Office of Section 508 Compliance: My-Health (44405): 508 Audit Report Findings Support

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#118311]
> [!NOTE]
> **QUICK SUMMARY:** _Parent ticket created to track Section 508 audit findings for the My-Health Website as part of Top Ten INTERnet Websites - June 2025 VASI._

## 508 Audit Office Scoring

**Overall Score:** `57.5`
- Critical: `2`
- High: `2`
- Medium: `1`
- Low: `0`
- Total: `5`

### About the 508 Office's Scoring

**A product with no defects has a score of 100**

- Each open 'Critical' defect lowers the score by 15.
- Each open 'High' defect lowers the score by 5.
- Each open 'Medium' defect lowers the score by 2.5.
- Each open 'Low' defect lowers the score by 1.
- Accessibility defects have no effect on the score.

### List of Defects:
The following examples are not and should not be interpreted as an exhaustive or complete list of all defects found in the software product. Examples were chosen to ease reproduction of the accessibility defect and to illustrate where multiple defects produce additional barriers to accessibility in combination. Stakeholders of the product should conduct internal testing to identify any additional instances of defects identified in this report. However, because these examples list the end-user experience of accessibility, defects identified may result from or be explained in terms of multiple criteria.
 
### Defect 6 | Critical | Created:
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships_
conveyed through presentation can be programmatically determined or are available in
text.
MyHealtheVet | Veterans Affairs
The list structure cannot be understood by assistive technology—the list structure is not
correct. For example, when arrow key is used to move focus from heading “Get
answers”, JAWS announce “list of 5 items”, but there are only 2 items under heading

“Get answers”. Similar issues occurred under heading “Call us” and “Visit a medical
center or regional office.” Please see attached screenshot. (New: 6/17/2025)

### Defect 7 | Medium | Created:
_WCAG 1.4.3 Contrast (Minimum) | The visual presentation of text and images of text_
has a contrast ratio of at least 4.5:1
MyHealtheVet | Veterans Affairs
Throughout the product, when interactive element receive focus, a visible focus
indicator is not sufficiently visible to be distinguished with low visual acuity. For example,
the color contrast of the focus indicator is 1.3:1 (#FACE00: #F0F0F0). Please see
attached screenshot. (New: 6/17/2025)

### Defect 8 | High | Created:
_WCAG 2.4.3 Focus Order | If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability._
MyHealtheVet | Veterans Affairs -&gt; Feedback button
When the “Feedback” modal pop up and navigating with arrow key, from “What task
were you trying to do today?” edit field, the focus moves out of modal to the parent
page. JAWS announces “Link Schedule a new appointment” on the parent page. New:
6/17/2025)

### Defect 9 | Critical | Created:
_WCAG 4.1.2 Name, Role, Value | For all user interface components (including but not
limited to: form elements, links and components generated by scripts), the name and
role can be programmatically determined; states, properties, and values that can be set
by the user can be programmatically set; and notification of changes to these items is
available to user agents, including assistive technologies._
MyHealtheVet | Veterans Affairs -&gt; Talk to the Veterans Crisis Line now button
As a JAWS user, when the “Talk to the Veterans Crisis Line now” modal dialog pop up,
JAWS announces “MyHealtheVet | Veterans Affairs - Google Chrome dialog.” The name
of dialog cannot be understood. (New: 6/17/2025)

### Defect 10 | High | Created:
_WCAG 2.4.3 Focus Order | If a Web page can be navigated sequentially and the
navigation sequences affect meaning or operation, focusable components receive focus
in an order that preserves meaning and operability._
MyHealtheVet | Veterans Affairs -&gt; Feedback button
As a JAWS user, when the “Talk to the Veterans Crisis Line now” modal dialog pop up,
the focus is on the “Call 988 and select 1” link. User might miss the information above
the “Call 988 and select 1” link. In addition, when the Feedback modal dialog pop up
(Feedback button), the focus is on the first interactive element (Close button). The focus
order for modals is not consistent. (New: 6/17/2025)

### Project Background

- Project Name: Top Ten INTERnet Websites - June 2025 VASI #: None  
- Audit Number: 44405 Audit Date: 6/17/2025
- Audit Scope: My-Health
- Product Type: Websites
- Project Status: Open
- Project Site: https://www.va.gov/my-health
- Project Version: 6/17/2025
- Release Recommendation: No
- Release Recommendation Comments: Product is non-conformant
- Link to 508 Compliance Application: [508 Compliance Application](apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/5f7da63e-384d-4ce4-8cc7-f3be6c60c363?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&sourcetime=2023-08-14%2017%3A50%3A55Z&RecordID=2989)
- 

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by lakper** [comment]
Created for tracking, closing ticket
